src/WebJobs.Script/Config/FunctionsHostingConfiguration.cs

```diff
@@ -19,6 +19,7 @@ internal class FunctionsHostingConfiguration : IFunctionsHostingConfiguration
 
         private DateTime _configTTL;
         private Dictionary<string, string> _configs;
+        private bool isConfigurationFileExists = true;
```

Comment:
fields should have an _ prefix. Also, recommend renaming to configurationFileExists